edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/PodExtensions.cs

```diff
@@ -61,15 +61,19 @@ static ReportedModuleStatus ConvertPodStatusToModuleStatus(Option<V1PodStatus> p
                                     else if (c.State.Terminated != null)
                                     {
                                         if (c.State.Terminated.ExitCode != 0)
+                                        {
```

Comment:
Why the formatting change?

Reply:
Rider suggestion of inconsistent brackets